crates/fluvio-hub-util/src/fvm/api/client.rs

```diff
@@ -43,16 +38,10 @@ impl Client {
 
     /// Builds the URL to the Hub API for fetching a [`PackageSet`] using the
     /// [`Client`]'s `api_url`.
-    fn make_fetch_package_set_url(
-        &self,
-        name: impl AsRef<str>,
-        channel: &Channel,
-        arch: &str,
-    ) -> Result<Url> {
+    fn make_fetch_package_set_url(&self, channel: &Channel, arch: &str) -> Result<Url> {
         let url = format!(
-            "{}hub/v1/fvm/pkgset/{name}/{channel}/{arch}",
+            "{}hub/v1/fvm/pkgset/{channel}?arch={arch}",
```

Comment:
Our URL now holds the architecture as a query parameter.

Reply:
That's good improvement but shouldn't that bump up version number since it's not compatible with V1?

Reply:
Since hub V1 is not released, my concern is addressed